src/styles/bloks/container.scss

```
@@ -0,0 +1,10 @@
+.container {
+  margin: 0 auto;
+  padding-top: 3px;
+  max-width: 942px;
+  &__block {
```

```suggestion
  max-width: 942px;

  &__block {
```
calculate it based on card width